code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -154,6 +153,8 @@
 	return TRUE
 
 /obj/item/clothing/mask/facehugger/proc/Impregnate(mob/living/target as mob)
```

Comment:
```suggestion
/obj/item/clothing/mask/facehugger/proc/Impregnate(mob/living/target)
```

Reply:
Done. Also discovered that `EyeBlind` is necessary since clothing-induced blindness doesn't mimic all the effects of the trait (i.e. you can still examine objects while blinded by a facehugger), so I readded that line.